db/migrate/20220920153113_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.integer :like_counter
+      t.integer :comment_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Hi. As shown in your ERD diagram, the Post table has a property text so kindly add it to your migrations
<img width="324" alt="image" src="https://user-images.githubusercontent.com/37901469/191341311-b731d227-6c10-4159-8423-3f46b3bc1410.png">